pyuvsim/analyticbeam.py

```diff
@@ -115,7 +115,15 @@ def efield_to_power(self):
         pol_strings = ['XX', 'XY', 'YX', 'YY']
         self.polarization_array = np.array([uvutils.polstr2num(ps.upper()) for ps in pol_strings])
 
-    def interp(self, az_array, za_array, freq_array, reuse_spline=None, spline_opts=None):
+    def interp(
+        self,
+        az_array,
+        za_array,
+        freq_array,
+        reuse_spline=None,
+        spline_opts=None,
+        check_azza_domain=None
```

Comment:
Should we not just use `**kwargs` in the signature and then just ignore them? I imagine there might be more additions to `interp()` in pyuvbeam over time, and we don't always want to have to remember to add the parameter to the analytic beam.

Reply:
sure, that's probably reasonable. Longer term this should move to pyuvdata and we can think better about how to handle these there.